### PR TITLE
Fix document download name

### PR DESCRIPTION
**Pull request by lribi**
Fix for #194

### CODE REVIEW

File: apps/documents/views.py [hunk 1]
```diff
@@ -174,9 +174,10 @@ def get(self, request, *args, **kwargs):
         # Serve file
         filename = unicodedata.normalize('NFKD', doc.original_filename) \
                               .encode('us-ascii', 'ignore')
-        attachment = not filename.lower().endswith('.pdf')
-        return sendfile(request, doc.document.path,
-                attachment=attachment, attachment_filename=filename)
+        attachment = "inline" if filename.lower().endswith('.pdf') == True else "attachment"
```
Comment (mattbaumann) [inline apps/documents/views.py]: == True kann weggelassen werden.

Comment (lribi) [reply apps/documents/views.py]: Danke

**Comment by The-Compiler** [comment]
I merged it locally (and resolved conflicts after my Python / Django upgrades) - so expected this to (finally) be merged soon! :tada: 

I also added some tests here: c3d9681b3d71e07eb71579b3cbb23356a8556190

**Comment by The-Compiler** [comment]
Thanks for the contribution and sorry for the long delay - merged now, and will show up on studentenportal.ch in the next couple of days. Stay tuned!

**Pull request merged**